src/pages/index.js

```diff
@@ -32,7 +32,7 @@ function HomepageHeader() {
               </div>}
             imageSrc={HeroImageUrl}
             isHero={true}
-            title="Welcome to the Teradata Developers Portal"
+            title="Welcome to the Teradata Developer Portal"
```

Comment:
Can you make sure this is pulling from translations

Reply:
Done
<Banner
            content={
              <div>
                <Typography scale="body1">{translate({ message: 'developers.title_description' })}</Typography> 
                <br/>
                <Link to="https://quickstarts.teradata.com/">
                  <Button
                    label="Learn more"
                    primary={true}
                    icon="fa fa-arrow-right"
                    trailingIcon={true}
                  />
                </Link>
              </div>}
            imageSrc={HeroImageUrl}
            isHero={true}
            title={translate({ message: 'developers.welcome_message' })}
          />